File: app/src/main/java/com/cradletrial/cradlevhtapp/view/ui/intro/MyIntroFragmentInteractionListener.java
package com.cradletrial.cradlevhtapp.view.ui.intro;

/**
 * For INTRO wizard
 * This interface must be implemented by activities that contain this
 * fragment to allow an interaction in this fragment to be communicated
 * to the activity and potentially other fragments contained in that
 * activity.
 */
public interface MyIntroFragmentInteractionListener {
    // Move on to next page
    void advanceToNextPage();

    // Set NEXT button being enabled/disabled
    void setNextButtonEnabled(boolean b);
}

File: app/src/main/java/com/cradle/neptune/view/ui/intro/MyIntroFragmentInteractionListener.kt
package com.cradle.neptune.view.ui.intro

/**
 * For INTRO wizard
 * This interface must be implemented by activities that contain this
 * fragment to allow an interaction in this fragment to be communicated
 * to the activity and potentially other fragments contained in that
 * activity.
 */
interface MyIntroFragmentInteractionListener {
    // Move on to next page
    fun advanceToNextPage()

    // Set NEXT button being enabled/disabled
    fun setNextButtonEnabled(isEnabled: Boolean)
}
